src/SpikeCore/SpikeCore.Web/SpikeCore.Web.csproj

```diff
@@ -1,11 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <UserSecretsId>aspnet-SpikeCore.Web-E201EA6B-FD51-4107-8B35-2258C27E66F3</UserSecretsId>
     <TypeScriptToolsVersion>2.8</TypeScriptToolsVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <LangVersion>latest</LangVersion>
```

Comment:
The language version was bumped because using an async main requires C# 7.1 (docs: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-1).